.gitignore

```diff
@@ -1 +1,2 @@
 .idea
+.env
```

Comment:
Is this necessary?

Reply:
yes, .env files should not be pushed to the public repo